### PR TITLE
lint: move imports to the top (pep8 1.6 new check)

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

File: sale_pricelist_restrict_product_in_so/product.py [hunk 1]
```diff
@@ -21,11 +21,10 @@
 ###############################################################################
 
 import logging
+from openerp.osv import orm
```
Comment (gurneyalex) [inline sale_pricelist_restrict_product_in_so/product.py:25]: can you keep the blank line separating stdlib imports from framework imports?

Comment (lepistone) [inline sale_pricelist_restrict_product_in_so/product.py:25]: done

**Comment by pedrobaeza** [comment]
:+1: 

This is due to a new check in PEP8 library.

**Comment by lepistone** [comment]
I opened OCA/maintainer-quality-tools#164.

**Comment by lepistone** [comment]
I changed the commit message, now I know it's due to pep8 1.6.

**Comment by yvaucher** [comment]
:+1:

**Comment by pedrobaeza** [comment]
As @gurneyalex's comments have been honoured, I merge.

**Pull request merged**